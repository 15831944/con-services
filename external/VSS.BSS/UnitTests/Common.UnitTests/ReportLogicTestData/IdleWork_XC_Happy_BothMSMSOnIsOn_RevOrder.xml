<?xml version="1.0" encoding="utf-8" standalone="no"?>
<ArrayOfInboundDeviceEvent xmlns="http://www.nighthawk.com/nighthawk/data/rptevent/2009/03"><InboundDeviceEvent NS0:type="InboundDeviceEventUtilization" xmlns:NS0="http://www.w3.org/2001/XMLSchema-instance"><AdditionalData NS0:nil="true"></AdditionalData><Address NS0:nil="true"></Address><Asset NS0:nil="true"></Asset><Description NS0:nil="true"></Description><Destination>NighthawkReport</Destination><Device><DeviceID NS0:nil="true"></DeviceID><DeviceTypeID>1</DeviceTypeID><ExtDeviceID>10</ExtDeviceID></Device><EventID>0</EventID><EventUTC>2009-05-27T11:40:20.5029649Z</EventUTC><EventTypeID>73</EventTypeID><ExtDriverID NS0:nil="true"></ExtDriverID><Position><ElevationFeet NS0:nil="true"></ElevationFeet><Heading NS0:nil="true"></Heading><Latitude>39.895992</Latitude><Longitude>-105.121051</Longitude><Mileage NS0:nil="true"></Mileage><SpeedMph NS0:nil="true"></SpeedMph></Position><ReceivedUTC>2009-05-27T11:40:21.5029649Z</ReceivedUTC><Site NS0:nil="true"></Site><IsOn>true</IsOn><MaxSpeed NS0:nil="true"></MaxSpeed><RuntimeSeconds>19000</RuntimeSeconds><RuntimeType>RuntimeMeter</RuntimeType><Sensor><Description NS0:nil="true"></Description><SensorName NS0:nil="true"></SensorName><SensorNumber NS0:nil="true"></SensorNumber></Sensor><StartStopUTC NS0:nil="true"></StartStopUTC></InboundDeviceEvent><InboundDeviceEvent NS0:type="InboundDeviceEventUtilization" xmlns:NS0="http://www.w3.org/2001/XMLSchema-instance"><AdditionalData NS0:nil="true"></AdditionalData><Address NS0:nil="true"></Address><Asset NS0:nil="true"></Asset><Description NS0:nil="true"></Description><Destination>NighthawkReport</Destination><Device><DeviceID NS0:nil="true"></DeviceID><DeviceTypeID>1</DeviceTypeID><ExtDeviceID>10</ExtDeviceID></Device><EventID>0</EventID><EventUTC>2009-05-27T11:40:20.5029649Z</EventUTC><EventTypeID>26</EventTypeID><ExtDriverID NS0:nil="true"></ExtDriverID><Position><ElevationFeet NS0:nil="true"></ElevationFeet><Heading NS0:nil="true"></Heading><Latitude>40.895992</Latitude><Longitude>-104.121051</Longitude><Mileage NS0:nil="true"></Mileage><SpeedMph NS0:nil="true"></SpeedMph></Position><ReceivedUTC>2009-05-27T11:40:21.5029649Z</ReceivedUTC><Site NS0:nil="true"></Site><IsOn>false</IsOn><MaxSpeed NS0:nil="true"></MaxSpeed><RuntimeSeconds NS0:nil="true"></RuntimeSeconds><RuntimeType>Ignition</RuntimeType><Sensor><Description NS0:nil="true"></Description><SensorName NS0:nil="true"></SensorName><SensorNumber NS0:nil="true"></SensorNumber></Sensor><StartStopUTC NS0:nil="true"></StartStopUTC></InboundDeviceEvent><InboundDeviceEvent NS0:type="InboundDeviceEventUtilization" xmlns:NS0="http://www.w3.org/2001/XMLSchema-instance"><AdditionalData NS0:nil="true"></AdditionalData><Address NS0:nil="true"></Address><Asset NS0:nil="true"></Asset><Description NS0:nil="true"></Description><Destination>NighthawkReport</Destination><Device><DeviceID NS0:nil="true"></DeviceID><DeviceTypeID>1</DeviceTypeID><ExtDeviceID>10</ExtDeviceID></Device><EventID>0</EventID><EventUTC>2009-05-27T11:10:20.5029649Z</EventUTC><EventTypeID>51</EventTypeID><ExtDriverID NS0:nil="true"></ExtDriverID><Position><ElevationFeet NS0:nil="true"></ElevationFeet><Heading NS0:nil="true"></Heading><Latitude>40.895992</Latitude><Longitude>-104.121051</Longitude><Mileage NS0:nil="true"></Mileage><SpeedMph NS0:nil="true"></SpeedMph></Position><ReceivedUTC>2009-05-27T11:10:21.5029649Z</ReceivedUTC><Site NS0:nil="true"></Site><IsOn>false</IsOn><MaxSpeed NS0:nil="true"></MaxSpeed><RuntimeSeconds NS0:nil="true"></RuntimeSeconds><RuntimeType>Sensor</RuntimeType><Sensor><Description NS0:nil="true"></Description><SensorName>Blade</SensorName><SensorNumber>1</SensorNumber></Sensor><StartStopUTC NS0:nil="true"></StartStopUTC></InboundDeviceEvent><InboundDeviceEvent NS0:type="InboundDeviceEventUtilization" xmlns:NS0="http://www.w3.org/2001/XMLSchema-instance"><AdditionalData NS0:nil="true"></AdditionalData><Address NS0:nil="true"></Address><Asset NS0:nil="true"></Asset><Description NS0:nil="true"></Description><Destination>NighthawkReport</Destination><Device><DeviceID NS0:nil="true"></DeviceID><DeviceTypeID>1</DeviceTypeID><ExtDeviceID>10</ExtDeviceID></Device><EventID>0</EventID><EventUTC>2009-05-27T10:40:20.5029649Z</EventUTC><EventTypeID>22</EventTypeID><ExtDriverID NS0:nil="true"></ExtDriverID><Position><ElevationFeet NS0:nil="true"></ElevationFeet><Heading NS0:nil="true"></Heading><Latitude>41.895992</Latitude><Longitude>-103.121051</Longitude><Mileage NS0:nil="true"></Mileage><SpeedMph NS0:nil="true"></SpeedMph></Position><ReceivedUTC>2009-05-27T10:40:21.5029649Z</ReceivedUTC><Site NS0:nil="true"></Site><IsOn>false</IsOn><MaxSpeed NS0:nil="true"></MaxSpeed><RuntimeSeconds NS0:nil="true"></RuntimeSeconds><RuntimeType>Motion</RuntimeType><Sensor><Description NS0:nil="true"></Description><SensorName NS0:nil="true"></SensorName><SensorNumber NS0:nil="true"></SensorNumber></Sensor><StartStopUTC NS0:nil="true"></StartStopUTC></InboundDeviceEvent><InboundDeviceEvent NS0:type="InboundDeviceEventUtilization" xmlns:NS0="http://www.w3.org/2001/XMLSchema-instance"><AdditionalData NS0:nil="true"></AdditionalData><Address NS0:nil="true"></Address><Asset NS0:nil="true"></Asset><Description NS0:nil="true"></Description><Destination>NighthawkReport</Destination><Device><DeviceID NS0:nil="true"></DeviceID><DeviceTypeID>1</DeviceTypeID><ExtDeviceID>10</ExtDeviceID></Device><EventID>0</EventID><EventUTC>2009-05-27T10:10:20.5029649Z</EventUTC><EventTypeID>51</EventTypeID><ExtDriverID NS0:nil="true"></ExtDriverID><Position><ElevationFeet NS0:nil="true"></ElevationFeet><Heading NS0:nil="true"></Heading><Latitude>42.895992</Latitude><Longitude>-102.121051</Longitude><Mileage NS0:nil="true"></Mileage><SpeedMph NS0:nil="true"></SpeedMph></Position><ReceivedUTC>2009-05-27T10:10:21.5029649Z</ReceivedUTC><Site NS0:nil="true"></Site><IsOn>true</IsOn><MaxSpeed NS0:nil="true"></MaxSpeed><RuntimeSeconds NS0:nil="true"></RuntimeSeconds><RuntimeType>Sensor</RuntimeType><Sensor><Description NS0:nil="true"></Description><SensorName>Blade</SensorName><SensorNumber>1</SensorNumber></Sensor><StartStopUTC NS0:nil="true"></StartStopUTC></InboundDeviceEvent><InboundDeviceEvent NS0:type="InboundDeviceEventUtilization" xmlns:NS0="http://www.w3.org/2001/XMLSchema-instance"><AdditionalData NS0:nil="true"></AdditionalData><Address NS0:nil="true"></Address><Asset NS0:nil="true"></Asset><Description NS0:nil="true"></Description><Destination>NighthawkReport</Destination><Device><DeviceID NS0:nil="true"></DeviceID><DeviceTypeID>1</DeviceTypeID><ExtDeviceID>10</ExtDeviceID></Device><EventID>0</EventID><EventUTC>2009-05-27T09:40:20.5029649Z</EventUTC><EventTypeID>21</EventTypeID><ExtDriverID NS0:nil="true"></ExtDriverID><Position><ElevationFeet NS0:nil="true"></ElevationFeet><Heading NS0:nil="true"></Heading><Latitude>42.895992</Latitude><Longitude>-102.121051</Longitude><Mileage NS0:nil="true"></Mileage><SpeedMph NS0:nil="true"></SpeedMph></Position><ReceivedUTC>2009-05-27T09:40:21.5029649Z</ReceivedUTC><Site NS0:nil="true"></Site><IsOn>true</IsOn><MaxSpeed NS0:nil="true"></MaxSpeed><RuntimeSeconds NS0:nil="true"></RuntimeSeconds><RuntimeType>Motion</RuntimeType><Sensor><Description NS0:nil="true"></Description><SensorName NS0:nil="true"></SensorName><SensorNumber NS0:nil="true"></SensorNumber></Sensor><StartStopUTC NS0:nil="true"></StartStopUTC></InboundDeviceEvent><InboundDeviceEvent NS0:type="InboundDeviceEventUtilization" xmlns:NS0="http://www.w3.org/2001/XMLSchema-instance"><AdditionalData NS0:nil="true"></AdditionalData><Address NS0:nil="true"></Address><Asset NS0:nil="true"></Asset><Description NS0:nil="true"></Description><Destination>NighthawkReport</Destination><Device><DeviceID NS0:nil="true"></DeviceID><DeviceTypeID>1</DeviceTypeID><ExtDeviceID>10</ExtDeviceID></Device><EventID>0</EventID><EventUTC>2009-05-27T09:10:20.5029649Z</EventUTC><EventTypeID>11</EventTypeID><ExtDriverID NS0:nil="true"></ExtDriverID><Position><ElevationFeet NS0:nil="true"></ElevationFeet><Heading NS0:nil="true"></Heading><Latitude>42.895992</Latitude><Longitude>-102.121051</Longitude><Mileage NS0:nil="true"></Mileage><SpeedMph NS0:nil="true"></SpeedMph></Position><ReceivedUTC>2009-05-27T09:10:21.5029649Z</ReceivedUTC><Site NS0:nil="true"></Site><IsOn>true</IsOn><MaxSpeed NS0:nil="true"></MaxSpeed><RuntimeSeconds>10000</RuntimeSeconds><RuntimeType>Ignition</RuntimeType><Sensor><Description NS0:nil="true"></Description><SensorName NS0:nil="true"></SensorName><SensorNumber NS0:nil="true"></SensorNumber></Sensor><StartStopUTC NS0:nil="true"></StartStopUTC></InboundDeviceEvent><InboundDeviceEvent NS0:type="InboundDeviceEventUtilization" xmlns:NS0="http://www.w3.org/2001/XMLSchema-instance"><AdditionalData NS0:nil="true"></AdditionalData><Address NS0:nil="true"></Address><Asset NS0:nil="true"></Asset><Description NS0:nil="true"></Description><Destination>NighthawkReport</Destination><Device><DeviceID NS0:nil="true"></DeviceID><DeviceTypeID>1</DeviceTypeID><ExtDeviceID>10</ExtDeviceID></Device><EventID>0</EventID><EventUTC>2009-05-27T09:10:20.5029649Z</EventUTC><EventTypeID>11</EventTypeID><ExtDriverID NS0:nil="true"></ExtDriverID><Position><ElevationFeet NS0:nil="true"></ElevationFeet><Heading NS0:nil="true"></Heading><Latitude>42.895992</Latitude><Longitude>-102.121051</Longitude><Mileage NS0:nil="true"></Mileage><SpeedMph NS0:nil="true"></SpeedMph></Position><ReceivedUTC>2009-05-27T09:10:21.5029649Z</ReceivedUTC><Site NS0:nil="true"></Site><IsOn>true</IsOn><MaxSpeed NS0:nil="true"></MaxSpeed><RuntimeSeconds>10000</RuntimeSeconds><RuntimeType>Ignition</RuntimeType><Sensor><Description NS0:nil="true"></Description><SensorName NS0:nil="true"></SensorName><SensorNumber NS0:nil="true"></SensorNumber></Sensor><StartStopUTC NS0:nil="true"></StartStopUTC></InboundDeviceEvent></ArrayOfInboundDeviceEvent>
