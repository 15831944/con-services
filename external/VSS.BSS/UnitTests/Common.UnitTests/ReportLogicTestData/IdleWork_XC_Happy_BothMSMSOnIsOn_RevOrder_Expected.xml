<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<ExpectedUtilizationEvents><UtilizationEvent><ID>0</ID><DeviceTypeID>0</DeviceTypeID><IsWorking>false</IsWorking><IsOn>false</IsOn><RuntimeEventTypeID>2</RuntimeEventTypeID><MeterSeconds NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></MeterSeconds><InsertUTC>0001-01-01T00:00:00</InsertUTC><EventUTC>0001-01-01T00:00:00</EventUTC><ReceivedUTC>0001-01-01T00:00:00</ReceivedUTC><SensorNumber NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></SensorNumber><EventDeviceTime>0001-01-01T00:00:00</EventDeviceTime></UtilizationEvent><UtilizationEvent><ID>0</ID><DeviceTypeID>0</DeviceTypeID><IsWorking>false</IsWorking><IsOn>false</IsOn><RuntimeEventTypeID>3</RuntimeEventTypeID><MeterSeconds NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></MeterSeconds><InsertUTC>0001-01-01T00:00:00</InsertUTC><EventUTC>0001-01-01T00:00:00</EventUTC><ReceivedUTC>0001-01-01T00:00:00</ReceivedUTC><SensorNumber NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></SensorNumber><EventDeviceTime>0001-01-01T00:00:00</EventDeviceTime></UtilizationEvent><UtilizationEvent><ID>0</ID><DeviceTypeID>0</DeviceTypeID><IsWorking>false</IsWorking><IsOn>false</IsOn><RuntimeEventTypeID>4</RuntimeEventTypeID><MeterSeconds NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></MeterSeconds><InsertUTC>0001-01-01T00:00:00</InsertUTC><EventUTC>0001-01-01T00:00:00</EventUTC><ReceivedUTC>0001-01-01T00:00:00</ReceivedUTC><SensorNumber NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></SensorNumber><EventDeviceTime>0001-01-01T00:00:00</EventDeviceTime></UtilizationEvent><UtilizationEvent><ID>0</ID><DeviceTypeID>0</DeviceTypeID><IsWorking>true</IsWorking><IsOn>true</IsOn><RuntimeEventTypeID>3</RuntimeEventTypeID><MeterSeconds NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></MeterSeconds><InsertUTC>0001-01-01T00:00:00</InsertUTC><EventUTC>0001-01-01T00:00:00</EventUTC><ReceivedUTC>0001-01-01T00:00:00</ReceivedUTC><SensorNumber NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></SensorNumber><EventDeviceTime>0001-01-01T00:00:00</EventDeviceTime></UtilizationEvent><UtilizationEvent><ID>0</ID><DeviceTypeID>0</DeviceTypeID><IsWorking>true</IsWorking><IsOn>true</IsOn><RuntimeEventTypeID>4</RuntimeEventTypeID><MeterSeconds NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></MeterSeconds><InsertUTC>0001-01-01T00:00:00</InsertUTC><EventUTC>0001-01-01T00:00:00</EventUTC><ReceivedUTC>0001-01-01T00:00:00</ReceivedUTC><SensorNumber NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></SensorNumber><EventDeviceTime>0001-01-01T00:00:00</EventDeviceTime></UtilizationEvent><UtilizationEvent><ID>0</ID><DeviceTypeID>0</DeviceTypeID><IsWorking>false</IsWorking><IsOn>true</IsOn><RuntimeEventTypeID>2</RuntimeEventTypeID><MeterSeconds NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></MeterSeconds><InsertUTC>0001-01-01T00:00:00</InsertUTC><EventUTC>0001-01-01T00:00:00</EventUTC><ReceivedUTC>0001-01-01T00:00:00</ReceivedUTC><SensorNumber NS1:nil="true" xmlns:NS1="http://www.w3.org/2001/XMLSchema-instance"></SensorNumber><EventDeviceTime>0001-01-01T00:00:00</EventDeviceTime></UtilizationEvent></ExpectedUtilizationEvents>